### DISCUSSION TITLE
Welcome to Fooocus-Control Discussions!

### DISCUSSION THREAD

**Discussion by fenneishi** [fenneishi/Fooocus-Control#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by outsami-9527** [comment]
Hello and thank you for such a great tool. Personally, I have been using stable diffusion for a while now.
Here are some personal questions and suggestions.
①fooocus-control can consider adding some easy to use control combinations, the current openpose seems to be separate, I looked at the downloaded model is like this, can't we just a dw_openpose_full solution?
② Is the existing candy duplicated with the official cpds? depth I like it very much. Can you add other models such as title, Scribble sofe edge, etc., or put a few common combinations in one, so as to save space and keep a clean interface.
③ If possible, I hope to consider adding third-party plugins such as face fusion and ultimate-upscale-for-automatic1111, the former is to change the face, the latter is to zoom in without memory overflow!
④ Increase the mask function of internal drawing, it has been proposed before, but the author of fooocus didn't adopt it.
https://github.com/lllyasviel/Fooocus/pull/529

I hope that some of my immature opinions will be taken into consideration. Thank you very much!

**Comment by fenneishi** [comment]
①fooocus-control can consider adding some easy to use control combinations, the current openpose seems to be separate, I looked at the downloaded model is like this, can't we just a dw_openpose_full solution?
![image](https://github.com/fenneishi/Fooocus-Control/assets/33345538/b12a770a-e340-42c6-9f52-e836d143fbc4)

**Comment by fenneishi** [comment]
* ② Is the existing candy duplicated with the official cpds? depth I like it very much. Can you add other models such as title, Scribble sofe edge, etc., or put a few common combinations in one, so as to save space and keep a clean interface.
---
* canny and cpds are both native to Fooocus, not provided by Fooocus-Control. See README.md for more development plans
* cpds is different from canny
![image](https://github.com/fenneishi/Fooocus-Control/assets/33345538/8cd1e95a-2019-4f7b-95a9-e4ef9746fc8f)

**Comment by fenneishi** [comment]
* ③ If possible, I hope to consider adding third-party plugins such as face fusion and ultimate-upscale-for-automatic1111, the former is to change the face, the latter is to zoom in without memory overflow!
* ④ Increase the mask function of internal drawing, it has been proposed before, but the author of fooocus didn't adopt it.
https://github.com/lllyasviel/Fooocus/pull/529
---
I'll evaluate it.

**Reply by outsami-9527** [reply]
thanks！
